opm/parser/eclipse/Parser/tests/ParseContextTests.cpp

```diff
@@ -82,6 +83,40 @@ BOOST_AUTO_TEST_CASE(TestUnkownKeyword) {
 }
 
 
+BOOST_AUTO_TEST_CASE(TEST_UNKNOWN_OPERATE) {
+    const char * deck =
+        "OPERATE\n"
+        "SWL    6* MULTX  PERMX 1.E10       / Temp:  SWL=1.E10*PERMX\n"
+        "SWL    6* MINLIM SWL   1.0         /\n"
+        "SWL    6* LOG10  SWL               / Temp:  SWL=log(1.E10*PERMX)\n"
+        "SWL    6* MULTA  SWL   -0.06  0.91 / Final: SWL=0.31-0.06*log(PERMX)\n"
+        "--SWCR 6* COPY   SWL               / SWCR=SWL\n"
+        "SWCR   6* MULTA  SWL   1.0    0.0  / SWCR=SWL+0.0 (+0.3)\n"
+        "SWCR   6* MAXLIM SWCR  0.7         / max(SWCR)=0.7\n"
+        "SGU    6* MULTA  SWL   -1.0   1.0  / SGU=1-SWL\n"
+        "/\n";
+
+    ParseContext parseContext;
+    Parser parser(false);
```

Comment:
For some reason, if parser is initialized with default keywords addDefaultKeywords
`Parser parser;`

the test fails.

Any idea why?